packages/radix-ui-themes/src/styles/animations.css

```diff
@@ -1,144 +1,93 @@
-@media (prefers-reduced-motion: no-preference) {
-  @keyframes slideUpAndFadeIn {
-    from {
-      opacity: 0;
-      transform: translateY(4px) scale(0.97);
-    }
-    to {
-      opacity: 1;
-      transform: translateY(0) scale(1);
-    }
+@keyframes rt-fade-in {
```

Comment:
Moved the keyframes definitions outside of the media query as you still might want to use them even with reduced motion preference. E.g. macOS does some animations with reduced motion, they are just milder.